lib/applicationinsights.json

```diff
@@ -3,6 +3,11 @@
   "role": {
     "name": "rpe-send-letter-service"
   },
+  "instrumentation": {
```

Comment:
The spring scheduling we use brings in its own telemetry. This is how it gets disabled as per here - https://learn.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-config#suppress-specific-autocollected-telemetry